client/components/app.jsx

```diff
@@ -1,15 +1,56 @@
 import React from 'react';
+import $ from 'jquery';
+import RelatedRestaurant from './RelatedRestaurant';
 
 class App extends React.Component {
   constructor() {
     super();
     this.state = {
+      data: [],
+      cuisine: '',
+      district: '',
+      currentRestaurant: '',
+      districts: {},
     };
   }
 
+  componentDidMount() {
+    const endpoint = '/api/restaurants/';
+    $.ajax({
+      url: endpoint,
+      method: 'GET',
+      success: (query) => {
+        const newDistricts = {};
+        for (let i = 0; i < query.length; i += 1) {
+          if (newDistricts[query[i].district] === undefined) {
+            newDistricts[query[i].district] = query[i].district;
+          }
+        }
+        this.setState({
+          data: query,
+          cuisine: query[0].cuisine,
+          district: query[0].district,
+          currentRestaurant: query[0].name,
+          districts: newDistricts,
+        });
+      },
+    });
+  }
+
   render() {
+    const { data } = this.state;
```

Comment:
You can combine them all in one:
const { data, cuisine, district, currentRestaurant, districts} = this.state